interface/orders/receive_hl7_results.inc.php

```diff
@@ -823,13 +823,16 @@ function receive_hl7_results(&$hl7, &$matchreq, $lab_id = 0, $direction = 'B', $
                     $in_orderid = intval($porow['procedure_order_id']);
                 }
 
-                if (!$in_orderid) {
-                    // Create order.
+                if (!$in_orderid || $orc) {
```

Comment:
Looks like there should be a global setting so we don't force this behavior on everyone. So something like this:

if (!$in_orderid || ($orc && $GLOBALS['gbl_force_new_order_for_each_resultsonly_report'])) {